Forta-Bots-Challenge-3/src/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const L1_DAI_CONTRACT_ADDRESS: string = "0x6B175474E89094C44Da98b954EedeAC495271d0F";
+export const L1_OPTIMISM_ESCROW_ADDRESS: string = "0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65";
+export const L1_ARBITRUM_ESCROW_ADDRESS: string = "0xA10c7CE4b876998858b1a9E12b10092229539400";
+export const OPTIMISM_DAI_ADDRESS: string = "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1";
+export const ARBITRUM_DAI_ADDRESS: string = "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1";
```

Comment:
Since they are the same value, they could be combined to a single constant. But keep a clear name like you have. Perhaps something like `OP_&_ARB_DAI_ADDRESS`? Or `L2_DAI_ADDRESS` with a comment above above saying `// Optimism & Arbitrum DAI address`? Just brainstorming.